test/bugsnex/plug_test.exs

```diff
@@ -60,10 +60,10 @@ defmodule Bugsnex.PlugTest do
   end
 
   test "build_plug_env/2" do
-    conn = conn(:get, "/bang?foo=bar")
+    conn = conn(:get, "/bang?foo=bar&password=password&password_confirmation=password")
     plug_env = %{request: Bugsnex.Plug.build_request_data(conn),
                  context: "/bang",
-                 params: %{"foo" => "bar"},
+                 params: %{"foo" => "bar", "password" => "[FILTERED]", "password_confirmation" => "[FILTERED]"},
```

Comment:
the code above does a lot of things that are not tested here (it avoids altering structs, it works for nested structures, it works for list of things...) which I think would all be nice to test here :) If you don't wanna do it, let me know and I/we will add the tests.